### PR TITLE
Correction to Zeuthen chicane input

### PR DESCRIPTION
**Pull request by cemitch99**
A correction to the bend parameters appearing in the Zeuthen chicane benchmark example, as pointed out by @zhangli28 [here](https://github.com/ECP-WarpX/impactx/issues/581#issuecomment-2079409274).

This change is small, and does not visibly affect the reduced_diagnostics output.  No update to the CI analysis script is required.

### CODE REVIEW

File: examples/chicane/run_chicane.py [hunk 1]
```diff
@@ -48,17 +48,18 @@
 
 # design the accelerator lattice
 ns = 25  # number of slices per ds in the element
-rc = 10.35  # bend radius (meters)
+rc = 10.3462283686195526  # bend radius (meters)
 psi = 0.048345620280243  # pole face rotation angle (radians)
+lb = 0.500194828041958  # projected bend length (meters)
```
Comment (n01r) [inline examples/chicane/run_chicane.py]: The comment says `projected bend length` but it is the arc length, no? The projected length would be the exact 0.5 m.

**Review by n01r — COMMENTED** [review]
Thanks, @cemitch99!
I have just a little comment about a comment string. :)

P.S.: Your PR still shows 8 commits but I think it is really only one commit and the rest is already in `development`, right? You could rebase to `development` and then the commit structure will be simpler. :)

File: examples/chicane/run_chicane.py (hunk 1 above)
Comment (cemitch99) [reply examples/chicane/run_chicane.py]: Yes.

File: examples/chicane/input_chicane.in [hunk 2]
```diff
@@ -26,15 +26,15 @@ lattice.elements = monitor sbend1 dipedge1 drift1 dipedge2 sbend2 drift2      \
...
+sbend1.rc = -10.3462283686195526
 
 drift1.type = drift
 drift1.ds = 5.0058489435  # projected length 5 m
 
 sbend2.type = sbend
-sbend2.ds = 0.50037       # projected length 0.5 m, angle 2.77 deg
-sbend2.rc = 10.35
+sbend2.ds = 0.500194828041958       # projected length 0.5 m, angle 2.77 deg
+sbend2.rc = 10.3462283686195526
```
Comment (ax3l) [inline examples/chicane/input_chicane.in:37]: If you want, we could use user-defined constants and the parser here to make it a bit clearer how those (projected) parameters come to play.
- https://impactx.readthedocs.io/en/latest/usage/parameters.html#running-cpp-parameters-parser

File: examples/chicane/run_chicane.py [hunk 3]
```diff
@@ -48,17 +48,18 @@
 
 # design the accelerator lattice
 ns = 25  # number of slices per ds in the element
-rc = 10.35  # bend radius (meters)
+rc = 10.3462283686195526  # bend radius (meters)
```
Comment (ax3l) [inline examples/chicane/run_chicane.py:51]: We could use Python math here to make it a bit clearer how those (projected) parameters come to play.

**Review by ax3l — APPROVED** [review]

**Pull request merged**